enterprise/micronaut/src/org/netbeans/modules/micronaut/resources/layer.xml

```diff
@@ -30,4 +30,17 @@
             </folder>
         </folder>
     </folder>
+    <folder name="Projects">
+        <folder name="org-netbeans-modules-maven">
+            <folder name="io.micronaut.build:micronaut-maven-plugin">
+                <folder name="Lookup">
+                    <file name="maven-project-actions.instance">
+                        <attr name="instanceOf" stringvalue="org.netbeans.spi.project.LookupProvider"/>
+                        <attr name="instanceCreate" methodvalue="org.netbeans.api.maven.MavenActions.forProjectLayer"/>
+                        <attr name="resource" stringvalue="nbres:/org/netbeans/modules/micronaut/resources/micronaut-actions.xml"/>
```

Comment:
Consider replacing this XML snippet with an annotation in the future.

Reply:
Logged as [NETBEANS-5694](https://issues.apache.org/jira/browse/NETBEANS-5694).